client/src/pages/MainMenu.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import styled from "styled-components";
```

Comment:
styled-components/**macro**?

Reply:
yep, for better debugging 🤘🚀 its shows the original class-names while inspecting